heketi-centos-ci-tests.sh

```diff
@@ -11,6 +11,11 @@ vagrant_env(){
  "$@"
 }
 
+# get os debug info
+uname -a
+uname -r
```

Comment:
You can remove uname -r. It is redundant.

Reply:
done